packages/node_modules/@webex/webex-core/src/lib/services/service-catalogs.js

```diff
@@ -41,6 +41,72 @@ export default class ServiceCatalogs {
     this.hosts = [];
   }
 
+  /**
+   * Filter the {@link ServiceCatalogs#hosts} array against their active states.
+   *
+   * @private
+   * @memberof ServiceCatalogs
+   * @param {Object} pto - The parameter transfer object.
+   * @property {boolean} [pto.active] - Filter to ignore service state.
+   * @returns {Array<ServiceHost>} - The filtered host array.
+   */
+  filterActive({active = true} = {}) {
```

Comment:
Are these `pto` objects generated somewhere else? Seems like overkill for one parameter. Also, when would you run it with true?